docs/source/reservations/existing-machines.rst

```diff
@@ -78,6 +81,17 @@ Deploying SkyPilot
       CONTEXT_NAME=mycluster  # Optional, sets the context name in the kubeconfig. Defaults to "default".
       sky local up --ips $IP_FILE --ssh-user $SSH_USER --ssh-key-path $SSH_KEY --context-name $CONTEXT_NAME
 
+   If passwordless sudo is not available:
+
+   .. code-block:: bash
+
+      IP_FILE=ips.txt
+      SSH_USER=username
+      SSH_KEY=path/to/ssh/key
+      PASSWORD=password
+      CONTEXT_NAME=mycluster  # Optional, sets the context name in the kubeconfig. Defaults to "default".
+      sky local up --ips $IP_FILE --ssh-user $SSH_USER --ssh-key-path $SSH_KEY --context-name $CONTEXT_NAME --password $PASSWORD
```

Comment:
To keep the docs short, can we put this in a tip box and assume passwordless sudo is the default? Something like:

```
.. tip::
If your cluster does not have passwordless sudo, specify the sudo password with the `--password` arg:

.. code-block:: bash
  sky local up --ips $IP_FILE --ssh-user $SSH_USER --ssh-key-path $SSH_KEY --context-name $CONTEXT_NAME --password $PASSWORD
```

Reply:
Same for the cleanup command below.

Reply:
Updated, thanks!